### PR TITLE
Update extra-docs-linting.yml

### PR DESCRIPTION
**Pull request by claudiol**
Adding fedora collection

# Description
- Fixes issue with CI failing 
RuntimeError: The command
| ansible-doc -vvv --metadata-dump --no-fail-on-errors amazon.aws ansible.builtin ansible.posix fedora.linux_system_roles infra.osbuild
returned exit status 1 with error output:
| ERROR! Cannot use supplied collection fedora.linux_system_roles: unable to locate collection fedora.linux_system_roles. unable to locate collection fedora.linux_system_roles

FIXES:  CI Workflow

## Type of change

What is it?

- [ x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update
- [ ] Refactor

## Checklist

- [ ] Added changelog fragment
- [ ] Tests exist for affected features covering positive and negative scenarios

### CODE REVIEW

**Review by itewk — APPROVED** [review]
lgtm. but i dont have merge permissions.

**Review by ericzolf — APPROVED** [review]
Looks good to me!

**Comment by ericzolf** [comment]
I'm assuming that all failed pipelines were failing before this merge so not caring. We should nevertheless look into getting the multiple pipelines back to green before we do too many more changes.

**Pull request merged**
